Gemfile

```diff
@@ -43,6 +43,7 @@ gem 'awesome_print'
 gem 'dotenv'
 gem 'dotenv-rails'
 gem 'httparty'
+gem 'gmaps4rails'
```

Comment:
Bundler/OrderedGems: Gems should be sorted in an alphabetical order within their section of the Gemfile. Gem `gmaps4rails` should appear before `httparty`.